apps/www/src/state/plugin/notifications.js

```diff
@@ -23,6 +23,15 @@ export const notifications = {
         state.backgroundTasks.delete(taskId)
         return state
       },
+      errorDialog(state, error) {
+        return { ...state, error }
+      },
+      hideError(state) {
+        return {
+          ...state,
+          error: undefined,
+        }
```

Comment:
The SHACL dialog is displayed when the `error` state is set and hidden when undefined